MetaMask/metamask-extension
# Fix #847 - Don't show account address on token pages

## Explanation

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/847

It's a bit misleading to see the user's address on a given token page, when one might expect that to be the token's contract address.  As a first step, we're removing the account address from individual token pages.

## Screenshots/Screencaps


https://github.com/MetaMask/metamask-extension/assets/46655/3109c6f9-ea18-48a3-ae0d-4096e813356a



## Manual Testing Steps

1.  From the home screen, see the AddressCopyButton present
2. Click into the native currency -- see no AddressCopyButton
3. Click into a token -- AddressCopyButton

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
